packages/rancher-vsphere/rancher-vsphere-cpi/charts/templates/daemonset.yaml

```diff
@@ -67,10 +67,10 @@ spec:
           value: "true"
         - key: node-role.kubernetes.io/control-plane
           effect: NoSchedule
-          value: "true"
+          operator: Exists
         - key: node-role.kubernetes.io/etcd
           effect: NoSchedule
```

Comment:
The issue says the effect for etcd should be NoExecute. I'm not sure whether that's true, but if you looked into that and decided it should remain as NoSchedule, may I know the reason?